plugins/repository-azure/build.gradle

```diff
@@ -94,6 +94,7 @@ thirdPartyAudit {
     // Optional and not enabled by Elasticsearch
     'com.google.common.util.concurrent.internal.InternalFutureFailureAccess',
     'com.google.common.util.concurrent.internal.InternalFutures',
+    'com.azure.core.http.HttpHeaderName',
```

Comment:
@kartg this is very suspicious, shouldn't we update `com.azure:azure-core` to `1.34.0`?

Reply:
Good point! I clearly wasn't thinking when i made this change 😅